components/src/core/drawer/drawer-body/nav-item/drawer-nav-item.component.ts

```diff
@@ -105,14 +105,14 @@ export class DrawerNavItemComponent extends StateListener implements Omit<Drawer
     @ViewChild('expandIcon', { static: false }) expandIconEl: ElementRef;
     @ViewChild('collapseIcon', { static: false }) collapseIconEl: ElementRef;
 
-    isEmpty = (el): boolean => isEmptyView(el);
+    isEmpty = (el: ElementRef): boolean => isEmptyView(el);
     isNestedItem: boolean;
     drawerOpen: boolean;
     hasChildren = false;
     depth: number;
     id: number;
 
-    constructor(drawerService: DrawerService, private readonly changeDetectorRef: ChangeDetectorRef) {
+    constructor(drawerService: DrawerService, changeDetectorRef: ChangeDetectorRef) {
```

Comment:
are we changing these from private->public to avoid a linting rule, or is it because they make more sense to be public?

Reply:
Each Drawer subsection component inherits a `drawerService` and `changeDetector` from it's base class.  I was messing around with the access modifiers but finally settled on marking these services as `protected`.